### PR TITLE
clean up s3 template from code that deals with gateway for the data storage path

### LINKED ISSUE #110: clean up s3 template from code that deal with gateway for the data storage path
Since we're not going to use the gateway at all to route the data traffic for s3/minio we can remove the code to keep the template as clean as possible

### PR DESCRIPTION
**Pull request by zaibon**
fixes #110

### CODE REVIEW

File: templates/s3/schema.capnp [hunk 1]
```diff
@@ -2,24 +2,18 @@
...
+    dataShards @2: Int32=16; # 0-stor data shards config
+    parityShards @3: Int32=4; # 0-stor parity shards config
+    storageType @4: StorageType; # s3 storage type
+    storageSize @5: UInt16; # s3 storage size
+    namespaces @6: List(Namespace); # namespace services created for s3. This is set by the template.
+    minioLogin @7: Text; # minio login
+    minioPassword @8: Text; # minio password
+    minioUrl @9: Text; # url to access minio on. This is set by the template.
+    nsPassword @10: Text; # Namespace password
+    vmIp @11: Text; # Vm ip retrieved from the gateway
```
Comment (serboctor) [inline templates/s3/schema.capnp]: you don't need the vmIp and vmPort if you're not using the storageNic anymore.

File: templates/s3/s3.py [hunk 2]
```diff
@@ -15,13 +15,14 @@
...
 
+
 class S3(TemplateBase):
     version = '0.0.1'
     template_name = "s3"
 
     def __init__(self, name=None, guid=None, data=None):
         super().__init__(name=name, guid=guid, data=data)
-        self._storage_network_enabled = False
+        # self._storage_network_enabled = False
```
Comment (serboctor) [inline templates/s3/s3.py]: Why are we not removing this all together?

Comment (zaibon) [reply templates/s3/s3.py]: yep forgot about it. thx

File: templates/s3/schema.capnp (hunk 1 above)
Comment (zaibon) [reply templates/s3/schema.capnp]: ok I'll remove

**Pull request merged**